shortfin/python/shortfin_apps/sd/simple_client.py

```diff
@@ -236,7 +236,7 @@ def main():
     args = p.parse_args()
 
     if not is_connected(args.host, args.port):
-        exit(3)
+        ...  # exit(3)
```

Comment:
What happened here? Want to log something or keep the exit?

Reply:
This change should be dropped